### PR TITLE
feat: apply the changes from `rc/v1.0.0-rc1`

### PR DESCRIPTION
**Pull request by zemyblue**
## Description

Apply the changes up to 69a291e10e139e03f693487146a1cc639f543d98(v1.0.0-rc7) of the rc/v1.0.0-rc0 branch.

## Checklist:

- [ ] I followed the [contributing guidelines](https://github.com/Finschia/finschia/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/finschia/blob/main/CODE_OF_CONDUCT.md).
- [ ] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: RELEASE_CHANGELOG.md [hunk 1]
```diff
@@ -36,94 +36,94 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [v0.7.0] - 2022-11-29
 
-This version base on [lbm-sdk v0.46.0](https://github.com/line/lbm-sdk/releases/tag/v0.46.0).
+This version base on [lbm-sdk v0.46.0](https://github.com/Finschia/finschia-sdk/releases/tag/v0.46.0).
```
Comment (jaeseung-bae) [inline RELEASE_CHANGELOG.md:39]:
```suggestion
This version base on [finschia-sdk v0.46.0](https://github.com/Finschia/finschia-sdk/releases/tag/v0.46.0).
```

File: RELEASE_CHANGELOG.md [hunk 2]
```diff
@@ -36,94 +36,94 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
 
 ### Features
-* [\#108](https://github.com/line/lbm/pull/108) Bump github.com/line/lbm-sdk from e19f863a8 to a389b6330
-* [\#110](https://github.com/line/lbm/pull/110) apply GovMint on x/foundation
-* [\#111](https://github.com/line/lbm/pull/111) Bump github.com/line/lbm-sdk from 66988a235 to 0.46.0-rc9
-* (build) [\#113](https://github.com/line/lbm/pull/113) enable to use libsodium version ostracon
+* [\#108](https://github.com/Finschia/finschia/pull/108) Bump github.com/line/lbm-sdk from e19f863a8 to a389b6330
+* [\#110](https://github.com/Finschia/finschia/pull/110) apply GovMint on x/foundation
+* [\#111](https://github.com/Finschia/finschia/pull/111) Bump github.com/line/lbm-sdk from 66988a235 to 0.46.0-rc9
+* (build) [\#113](https://github.com/Finschia/finschia/pull/113) enable to use libsodium version ostracon
```
Comment (jaeseung-bae) [inline RELEASE_CHANGELOG.md:45]:
```suggestion
* [\#108](https://github.com/Finschia/finschia/pull/108) Bump github.com/Finschia/finschia-sdk from e19f863a8 to a389b6330
* [\#110](https://github.com/Finschia/finschia/pull/110) apply GovMint on x/foundation
* [\#111](https://github.com/Finschia/finschia/pull/111) Bump github.com/Finschia/finschia-sdk from 66988a235 to 0.46.0-rc9
* (build) [\#113](https://github.com/Finschia/finschia/pull/113) enable to use libsodium version ostracon
```

File: RELEASE_CHANGELOG.md [hunk 3]
```diff
@@ -36,94 +36,94 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
 
 ### Improvements
-* (app) [\#114](https://github.com/line/lbm/pull/114) change the default compile setting to support ledger
+* (app) [\#114](https://github.com/Finschia/finschia/pull/114) change the default compile setting to support ledger
 
 
 ## [v0.6.0]
 
-This version based on [lbm-sdk v0.46.0-rc8](https://github.com/line/lbm-sdk/releases/tag/v0.46.0-rc8)
+This version based on [lbm-sdk v0.46.0-rc8](https://github.com/Finschia/finschia-sdk/releases/tag/v0.46.0-rc8)
```
Comment (jaeseung-bae) [inline RELEASE_CHANGELOG.md:61]:
```suggestion
This version based on [finschia-sdk v0.46.0-rc8](https://github.com/Finschia/finschia-sdk/releases/tag/v0.46.0-rc8)
```

File: RELEASE_CHANGELOG.md [hunk 4]
```diff
@@ -36,94 +36,94 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
 
 ### Bug Fixes
-* (app) [\#96](https://github.com/line/lbm/pull/96) fix the bug not setting `iavl-cache-size` value of the `app.toml`
-* (baseapp) [\#97](https://github.com/line/lbm/pull/97) fix max gas validation bug of lbm-sdk
+* (app) [\#96](https://github.com/Finschia/finschia/pull/96) fix the bug not setting `iavl-cache-size` value of the `app.toml`
+* (baseapp) [\#97](https://github.com/Finschia/finschia/pull/97) fix max gas validation bug of lbm-sdk
```
Comment (jaeseung-bae) [inline RELEASE_CHANGELOG.md:65]:
```suggestion
* (baseapp) [\#97](https://github.com/Finschia/finschia/pull/97) fix max gas validation bug of finschia-sdk
```

File: RELEASE_CHANGELOG.md [hunk 5]
```diff
@@ -36,94 +36,94 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
 
 ### Breaking Changes
-* (app) [\#87](https://github.com/line/lbm/pull/87) remove unused modules from app
-* (lbm-sdk) [\#95](https://github.com/line/lbm/pull/95) apply the changes of lbm-sdk v0.46.0-rc8
+* (app) [\#87](https://github.com/Finschia/finschia/pull/87) remove unused modules from app
+* (lbm-sdk) [\#95](https://github.com/Finschia/finschia/pull/95) apply the changes of lbm-sdk v0.46.0-rc8
```
Comment (jaeseung-bae) [inline RELEASE_CHANGELOG.md:69]:
```suggestion
* (finschia-sdk) [\#95](https://github.com/Finschia/finschia/pull/95) apply the changes of finschia-sdk v0.46.0-rc8
```

File: RELEASE_CHANGELOG.md [hunk 6]
```diff
@@ -36,94 +36,94 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
-* (build) [\#89](https://github.com/line/lbm/pull/89) upgrade golang to 1.18
+* (ci) [\#80](https://github.com/Finschia/finschia/pull/80) remove stale github action
+* (ci) [\#83](https://github.com/Finschia/finschia/pull/83) enable tests on CI
+* (build) [\#89](https://github.com/Finschia/finschia/pull/89) upgrade golang to 1.18
 
 
 ## [v0.5.0]
 
-This version based on [lbm-sdk v0.46.0-rc6](https://github.com/line/lbm-sdk/releases/tag/v0.46.0-rc6)
+This version based on [lbm-sdk v0.46.0-rc6](https://github.com/Finschia/finschia-sdk/releases/tag/v0.46.0-rc6)
```
Comment (jaeseung-bae) [inline RELEASE_CHANGELOG.md:79]:
```suggestion
This version based on [finschia-sdk v0.46.0-rc6](https://github.com/Finschia/finschia-sdk/releases/tag/v0.46.0-rc6)
```

File: RELEASE_CHANGELOG.md [hunk 7]
```diff
@@ -36,94 +36,94 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
-* (x/collection) [\#86](https://github.com/line/lbm/pull/86) add omitted cli commands on x/collection and fix Query/Balance
-* (x/collection) [\#90](https://github.com/line/lbm/pull/90) fix bugs in x/collection MsgModify
+* (command) [\#81](https://github.com/Finschia/finschia/pull/81) add wrong address to genesis file in add-genesis-account command
+* (x/collection) [\#86](https://github.com/Finschia/finschia/pull/86) add omitted cli commands on x/collection and fix Query/Balance
+* (x/collection) [\#90](https://github.com/Finschia/finschia/pull/90) fix bugs in x/collection MsgModify
 
 
 ## [v0.4.0]
 
 This version based on lbm-sdk v0.46.0-rc2
```
Comment (jaeseung-bae) [inline RELEASE_CHANGELOG.md:96]:
```suggestion
This version based on finschia-sdk v0.46.0-rc2
```

File: RELEASE_CHANGELOG.md [hunk 8]
```diff
@@ -134,17 +134,17 @@ This version based on lbm-sdk v0.46.0-rc2
 * (sdk) Use fastcache for inter block cache and iavl cache
 * (sdk) Enable signature verification cache
 * (ostracon) Apply asynchronous receiving reactor
-* (sdk) [\#21](https://github.com/line/lbm/pull/21) Use lbm-sdk v0.43.0
+* (sdk) [\#21](https://github.com/Finschia/finschia/pull/21) Use lbm-sdk v0.43.0
```
Comment (jaeseung-bae) [inline RELEASE_CHANGELOG.md:137]:
```suggestion
* (sdk) [\#21](https://github.com/Finschia/finschia/pull/21) Use finschia-sdk v0.43.0
```

File: RELEASE_CHANGELOG.md [hunk 9]
```diff
@@ -134,17 +134,17 @@ This version based on lbm-sdk v0.46.0-rc2
...
+* (sdk) (auth) [\#16](https://github.com/Finschia/lfb/pull/16) Introduce sig block height for the new replay protection
+* (ostracon/sdk) [\#26](https://github.com/Finschia/lfb/pull/26) Use vrf-based consensus, address string treatment
+* (global) [\#10](https://github.com/Finschia/finschia/pull/10) Re-brand lfb to lbm
 
 ### Build, CI
-* (build) [\#25](https://github.com/line/lbm/pull/25) Fix localnet-start
+* (build) [\#25](https://github.com/Finschia/finschia/pull/25) Fix localnet-start
 
 ## [gaia v4.0.4] - 2021-03-15
 Initial lbm is based on the tendermint v0.34.9+, cosmos-sdk v0.42.0+, gaia v4.0.4
```
Comment (jaeseung-bae) [inline RELEASE_CHANGELOG.md:150]:
```suggestion
Initial finschia is based on the tendermint v0.34.9+, cosmos-sdk v0.42.0+, gaia v4.0.4
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -37,25 +37,118 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
+
+
+## [v1.0.0-rc5](https://github.com/Finschia/finschia/releases/tag/v1.0.0-rc5) - 2023-04-13
+
+### Improvements
+* (wasmd) [\#171](https://github.com/Finschia/finschia/pull/171) bump up wasmd from v0.1.2-0.20230403061848-514953c0b244 to v0.1.2
+
+### Bug Fixes
+* (lbm-sdk) [\#168](https://github.com/Finschia/finschia/pull/168) Bump github.com/line/lbm-sdk from v0.47.0-rc4 to v0.47.0-rc4.0.20230410115856-b8421116b3f2
+* (lbm-sdk) [\#172](https://github.com/Finschia/finschia/pull/172) Bump github.com/line/lbm-sdk from v0.47.0-rc4.0.20230410115856-b8421116b3f2 to v0.47.0-rc5
```
Comment (jaeseung-bae) [inline CHANGELOG.md:72]:
```suggestion
* (lbm-sdk) [\#168](https://github.com/Finschia/finschia/pull/168) Bump github.com/Finschia/finschia-sdk from v0.47.0-rc4 to v0.47.0-rc4.0.20230410115856-b8421116b3f2
* (lbm-sdk) [\#172](https://github.com/Finschia/finschia/pull/172) Bump github.com/Finschia/finschia-sdk from v0.47.0-rc4.0.20230410115856-b8421116b3f2 to v0.47.0-rc5
```

File: CHANGELOG.md [hunk 11]
```diff
@@ -37,25 +37,118 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
+
+
+## [v1.0.0-rc4](https://github.com/Finschia/finschia/releases/tag/v1.0.0-rc4) - 2023-04-08
+
+### Features
+
+### Improvements
+
+### Bug Fixes
+* (lbm-sdk) [\#167](https://github.com/Finschia/finschia/pull/167) Bump github.com/line/lbm-sdk from v0.47.0-rc3 to v0.47.0-rc4
```
Comment (jaeseung-bae) [inline CHANGELOG.md:86]:
```suggestion
* (lbm-sdk) [\#167](https://github.com/Finschia/finschia/pull/167) Bump github.com/Finschia/finschia-sdk from v0.47.0-rc3 to v0.47.0-rc4
```

File: CHANGELOG.md [hunk 12]
```diff
@@ -37,25 +37,118 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
+
+### Build, CI
+
+### Docs
+
+
+## [v1.0.0-rc3](https://github.com/Finschia/finschia/releases/tag/v1.0.0-rc3) - 2023-04-03
+
+### Bug Fixes
+* (lbm-sdk) [\#162](https://github.com/Finschia/finschia/pull/162) Bump github.com/line/lbm-sdk from v0.47.0-rc2 to v0.47.0-rc3
```
Comment (jaeseung-bae) [inline CHANGELOG.md:98]:
```suggestion
* (lbm-sdk) [\#162](https://github.com/Finschia/finschia/pull/162) Bump github.com/Finschia/finschia-sdk from v0.47.0-rc2 to v0.47.0-rc3
```

File: CHANGELOG.md [hunk 13]
```diff
@@ -37,25 +37,118 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
+
+### Docs
+* (doc)[\#165](https://github.com/Finschia/finschia/pull/165) update license notice and code_of_conduct
+
+
+## [v1.0.0-rc2](https://github.com/Finschia/finschia/releases/tag/v1.0.0-rc2) - 2023-03-29
+
+### Improvements
+* (x/wasmd) [\#158](https://github.com/Finschia/finschia/pull/158) bump up wasmd version to v0.1.0
+* (lbm-sdk) [\#159](https://github.com/Finschia/finschia/pull/159) Bump github.com/line/lbm-sdk from v0.47.0-rc1 to v0.47.0-rc2
```
Comment (jaeseung-bae) [inline CHANGELOG.md:114]:
```suggestion
* (lbm-sdk) [\#159](https://github.com/Finschia/finschia/pull/159) Bump github.com/Finschia/finschia-sdk from v0.47.0-rc1 to v0.47.0-rc2
```

File: CHANGELOG.md [hunk 14]
```diff
@@ -37,25 +37,118 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
+### Docs
+* (doc) [\#156](https://github.com/Finschia/finschia/pull/156) modify broken link or typo doc file and add issue and pr template
+
+
+## [v1.0.0-rc1](https://github.com/Finschia/finschia/releases/tag/v1.0.0-rc1) - 2023-03-24
+
+### Features
+* (build) [\#150](https://github.com/Finschia/finschia/pull/150) Modify the Makefile to build release bundles
+* (build) [\#153](https://github.com/Finschia/finschia/pull/153) rename cli name to `fnsad`
+* (lbm-sdk) [\#154](https://github.com/Finschia/finschia/pull/154) Bump github.com/line/lbm-sdk from v0.47.0-alpha1.0.20230214070148-11966d123415 to v0.47.0-rc1
```
Comment (jaeseung-bae) [inline CHANGELOG.md:128]:
```suggestion
* (lbm-sdk) [\#154](https://github.com/Finschia/finschia/pull/154) Bump github.com/Finschia/finschia-sdk from v0.47.0-alpha1.0.20230214070148-11966d123415 to v0.47.0-rc1
```

File: CHANGELOG.md [hunk 15]
```diff
@@ -37,25 +37,118 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
+* (x/wasmd) [\#147](https://github.com/Finschia/finschia/pull/147) update wasmd version
+
+
+## [v1.0.0-rc0](https://github.com/Finschia/finschia/releases/tag/v1.0.0-rc0) - 2023-02-16
+
+### Features
+* (build) [\#126](https://github.com/Finschia/finschia/pull/126) Automatically generates release note and binaries
+* (x/wasmd) [\#129](https://github.com/Finschia/finschia/pull/129) chore: apply detached x/wasmd
+* (build) [\#130](https://github.com/Finschia/finschia/pull/130) Add a release build for the linux/arm64, darwin/amd64, and darwin/arm64 platform
+* (lbm-sdk) [\#137](https://github.com/Finschia/finschia/pull/137) Bump line/lbm-sdk to 6c84a4cffa
```
Comment (jaeseung-bae) [inline CHANGELOG.md:140]:
```suggestion
* (lbm-sdk) [\#137](https://github.com/Finschia/finschia/pull/137) Bump Finschia/finschia-sdk to 6c84a4cffa
```

File: CHANGELOG.md [hunk 16]
```diff
@@ -37,25 +37,118 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
+* (x/collection,token) [\#138](https://github.com/Finschia/finschia/pull/138) Add x/token and x/collection
+* (ibc-go) [\#140](https://github.com/Finschia/finschia/pull/140) apply ibc-go
+* (x/wasmplus) [\#141](https://github.com/Finschia/finschia/pull/141) change wasm module to wrapped `x/wasmplus`
+* (lbm-sdk) [\#144](https://github.com/Finschia/finschia/pull/144) bump line/lbm-sdk v0.47.0-alpha1 (11966d1234155ebef20b64f2ae7a905beffdb33f)
```
Comment (jaeseung-bae) [inline CHANGELOG.md:144]:
```suggestion
* (finschia-sdk) [\#144](https://github.com/Finschia/finschia/pull/144) bump Finschia/finschia-sdk v0.47.0-alpha1 (11966d1234155ebef20b64f2ae7a905beffdb33f) 
```

File: RELEASE_CHANGELOG.md (hunk 2 above)
Comment (zemyblue) [reply RELEASE_CHANGELOG.md:45]: This is commit message, not a url. So we didn't modify it.

**Comment by zemyblue** [comment]
@jaeseung-bae , I think it's better to change to another PR. Because this pr is just backport.

File: CHANGELOG.md (hunk 15 above)
Comment (zemyblue) [reply CHANGELOG.md:140]: Since this is just a commit message, I'd prefer not to modify it.

File: CHANGELOG.md (hunk 16 above)
Comment (zemyblue) [reply CHANGELOG.md:144]: ditto

File: RELEASE_CHANGELOG.md (hunk 4 above)
Comment (zemyblue) [reply RELEASE_CHANGELOG.md:65]: This is commit message, not a url. So we didn't modify it.

**Review by jaeseung-bae — APPROVED** [review]

**Review by loin3 — APPROVED** [review]

**Pull request merged**